tests/h/h_api/conftest.py

```diff
@@ -59,12 +59,18 @@ def create_group_membership_body():
 
 @pytest.fixture
 def user_attributes(upsert_user_body):
-    return upsert_user_body["data"]["attributes"]
+    return dict(
+        upsert_user_body["data"]["attributes"],
+        **upsert_user_body["data"]["meta"]["query"]
```

Comment:
Merge the query into the attrs to get the items we need to call the create method with